lib/analytics.js

```diff
@@ -791,12 +821,43 @@ Analytics.prototype._invoke = function(method, facade) {
                 result = new Facade(result);
               }
 
-              metrics.increment('analytics_js.integration.invoke', {
-                method: method,
-                integration_name: integration.name
-              });
+              // apply destination middlewares
+              // Apply any integration middlewares that exist, then invoke the integration with the result.
+              if (self._destinationMiddlewares[integration.name]) {
+                self._destinationMiddlewares[integration.name].applyMiddlewares(
+                  facadeCopy,
+                  integration.name,
+                  function(result) {
+                    // A nullified payload should not be sent to an integration.
+                    if (result === null) {
```

Comment:
is a nullified payload something unexpected that we should be worried about? If so, should we report metrics on them?

Reply:
I think this is the case where the user explicitly wants to drop the event, so its a feature really

Reply:
gotcha! thanks for the clarification :D